packages/ui/src/components/AppRoot/AppRoot.tsx

```diff
@@ -0,0 +1,10 @@
+import classNames from 'classnames';
+import { HTMLAttributes } from 'react';
+
+import styles from './AppRoot.module.css';
+
+export interface AppRootProps extends HTMLAttributes<HTMLDivElement> {}
```

Comment:
I think it is better to use `ComponentType<'div'>` for singularity.

For the same reason, I think its better to not export these interfaces and in other files do something like `ComponentType<typeof AppRoot>` since its a recommended react approach IMO. Overall this would be more logical and easier to understand

Reply:
My bad, i mean `ComponentProps`